internal/cli/runner_profile_set.go

```diff
@@ -186,37 +186,37 @@ func (c *OnDemandRunnerConfigApplyCommand) Run(args []string) int {
 	})
 	if err != nil {
 		c.ui.Output(
-			"Error upserting on-demand runner config: %s", clierrors.Humanize(err),
+			"Error upserting runner profile: %s", clierrors.Humanize(err),
 			terminal.WithErrorStyle(),
 		)
 		return 1
 	}
 
 	if updated {
-		s.Update("On-Demand Runner configuration updated")
+		s.Update("Runner profile updated")
 	} else {
-		s.Update("On-Demand Runner configuration created")
+		s.Update("Runner profile created")
 	}
 	s.Done()
 
 	return 0
 }
 
-func (c *OnDemandRunnerConfigApplyCommand) Flags() *flag.Sets {
+func (c *RunnerProfileSetCommand) Flags() *flag.Sets {
 	return c.flagSet(0, func(sets *flag.Sets) {
 		f := sets.NewSet("Command Options")
 
 		f.StringVar(&flag.StringVar{
 			Name:    "name",
 			Target:  &c.flagName,
 			Default: "",
-			Usage:   "The name of an existing on-demand runner to update.",
+			Usage:   "The name of an existing runner profile to update.",
 		})
 
 		f.StringVar(&flag.StringVar{
 			Name:    "oci-url",
 			Target:  &c.flagOCIUrl,
-			Default: "hashicorp/waypoint:stable",
+			Default: "hashicorp/waypoint:stable", // TODO: update to published prod ODR image before 0.6.0 release
```

Comment:
TODO?

Reply:
Ah, that's unrelated to this change - just something I noticed that we'll need to get to before the release. I figure it'll be `hashicorp/waypoint-odr`, but I don't see us referencing that anywhere - it seems like we don't have the circleCI conf yet to publish it?